language/move-prover/bytecode-to-boogie/tests/translator_tests.rs

```diff
@@ -34,55 +34,11 @@ fn test_struct() {
     test(NO_VERIFY, &["test_mvir/test-struct.mvir"]);
 }
 
-#[test]
-fn test_access_path() {
-    test(
-        &["--native-stubs"],
-        &[
-            &std_mvir("vector"),
-            &std_mvir("u64_util"),
-            &std_mvir("address_util"),
-            &std_mvir("hash"),
-            &std_mvir("libra_coin"),
-            &std_mvir("libra_time"),
-            &std_mvir("libra_transaction_timeout"),
-            &std_mvir("libra_account"),
-            "test_mvir/test-access-path.mvir",
```

Comment:
The purpose of this test was to document an issue in the IR parser that is not present in the source language parser. Didn't seem worth keeping around.